sns_ik_lib/src/fosns_velocity_ik.cpp

```diff
@@ -102,49 +102,49 @@ Scalar FOSNSVelocityIK::getJointVelocity(VectorD *jointVelocity,
 
 //#define LOG_ACTIVE
 
-Scalar FOSNSVelocityIK::SNSsingle(int priority,
-                                  const VectorD &higherPriorityJointVelocity,
-                                  const MatrixD &higherPriorityNull,
-                                  const MatrixD &jacobian,
-                                  const VectorD &task,
-                                  VectorD *jointVelocity,
-                                  MatrixD *nullSpaceProjector)
+double FOSNSVelocityIK::SNSsingle(int priority,
+                                  const Eigen::VectorXd &higherPriorityJointVelocity,
+                                  const Eigen::MatrixXd &higherPriorityNull,
+                                  const Eigen::MatrixXd &jacobian,
+                                  const Eigen::VectorXd &task,
+                                  Eigen::VectorXd *jointVelocity,
+                                  Eigen::MatrixXd *nullSpaceProjector)
 {
   //INITIALIZATION
-  MatrixD JPinverse;  //(J_k P_{k-1})^#
-  Array<Scalar, Dynamic, 1> a, b;  // used to compute the task scaling factor
+  Eigen::MatrixXd JPinverse;  //(J_k P_{k-1})^#
+  Array<double, Dynamic, 1> a, b;  // used to compute the task scaling factor
```

Comment:
Fine to add the Eigen namespace to the other lines, but add it here as well to be consistent.

Reply:
good suggestion - I'll fix this too.